package.json

```diff
@@ -38,8 +39,7 @@
   "devDependencies": {
     "@adfinis-sygroup/eslint-config": "1.5.0",
     "@adfinis-sygroup/semantic-release-config": "3.4.0",
-    "@babel/helper-simple-access": "7.19.4",
-    "@babel/plugin-proposal-object-rest-spread": "7.18.9",
+    "@babel/core": "^7.0.0",
```

Comment:
```bash
WARN  Issues with peer dependencies found
.
├─┬ @adfinis-sygroup/semantic-release-config 3.4.0
│ └─┬ @commitlint/cli 16.0.2
│   └─┬ @commitlint/load 16.0.0
│     └─┬ cosmiconfig-typescript-loader 1.0.3
│       ├── ✕ missing peer @types/node@"*"
│       └─┬ ts-node 10.9.1
│         └── ✕ missing peer @types/node@"*"
├─┬ @ember/render-modifiers 2.0.4
│ └─┬ ember-modifier-manager-polyfill 1.2.0
│   └─┬ ember-compatibility-helpers 1.2.5
│     └─┬ babel-plugin-debug-macros 0.2.0
│       └── ✕ missing peer @babel/core@^7.0.0-beta.42
├─┬ @glimmer/component 1.1.2
│ └─┬ ember-cli-typescript 3.0.0
│   └─┬ @babel/plugin-transform-typescript 7.5.5
│     ├── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/helper-create-class-features-plugin 7.19.0
│     │ └── ✕ missing peer @babel/core@^7.0.0
│     └─┬ @babel/plugin-syntax-typescript 7.16.7
│       └── ✕ missing peer @babel/core@^7.0.0-0
├─┬ ember-data 3.28.0
│ └─┬ @ember-data/adapter 3.28.0
│   └─┬ @ember-data/private-build-infra 3.28.0
│     ├─┬ @babel/plugin-transform-block-scoping 7.19.4
│     │ └── ✕ missing peer @babel/core@^7.0.0-0
│     └─┬ babel-plugin-debug-macros 0.3.4
│       └── ✕ missing peer @babel/core@^7.0.0
├─┬ ember-load-initializers 2.1.2
│ └─┬ ember-cli-typescript 2.0.2
│   ├─┬ @babel/plugin-proposal-class-properties 7.18.6
│   │ └── ✕ missing peer @babel/core@^7.0.0-0
│   └─┬ @babel/plugin-transform-typescript 7.4.5
│     └── ✕ missing peer @babel/core@^7.0.0-0
├─┬ ember-power-select 6.0.1
│ └─┬ ember-concurrency-decorators 2.0.3
│   └─┬ ember-cli-typescript 3.1.4
│     ├─┬ @babel/plugin-proposal-nullish-coalescing-operator 7.18.6
│     │ ├── ✕ missing peer @babel/core@^7.0.0-0
│     │ └─┬ @babel/plugin-syntax-nullish-coalescing-operator 7.8.3
│     │   └── ✕ missing peer @babel/core@^7.0.0-0
│     ├─┬ @babel/plugin-proposal-optional-chaining 7.18.9
│     │ ├── ✕ missing peer @babel/core@^7.0.0-0
│     │ └─┬ @babel/plugin-syntax-optional-chaining 7.8.3
│     │   └── ✕ missing peer @babel/core@^7.0.0-0
│     └─┬ @babel/plugin-transform-typescript 7.8.7
│       └── ✕ missing peer @babel/core@^7.0.0-0
└─┬ ember-source 3.28.0
  └─┬ @babel/plugin-transform-object-assign 7.16.0
    └── ✕ missing peer @babel/core@^7.0.0-0
Peer dependencies that should be installed:
  @babel/core@">=7.0.0 <8.0.0"  @types/node@"*"
```
And after some short research found this advice from the maker of `pnpm` and just installed it.

Reply:
I'd prefer using something like this:
```json
{
  "pnpm": {
    "peerDependencyRules": {
      "ignoreMissing": ["@babel/core", "@types/node"]
    }
  }
}
```
instead of adding random unneeded dependencies..

Reply:
:+1: Adapted.